apps/portals/challenges/src/config/routesConfig.tsx

```diff
@@ -21,65 +18,56 @@ const routes: RouteObject[] = [
-                          titleLinkConfig: challengeTitleLinkConfig,
-                        }}
-                      />
-                    ),
-                  },
-                  {
-                    label: 'Upcoming',
-                    element: (
-                      <CardContainerLogic
-                        sql={`${challengeProjectsSql} where Status='Upcoming'`}
-                        cardConfiguration={{
-                          ...challengeCardConfiguration,
-                          titleLinkConfig: challengeTitleLinkConfig,
-                        }}
-                      />
-                    ),
-                  },
-                  {
-                    label: 'Completed',
-                    element: (
-                      <CardContainerLogic
-                        sql={`${challengeProjectsSql} where Status='Closed'`}
-                        cardConfiguration={{
-                          ...challengeCardConfiguration,
-                          titleLinkConfig: challengeTitleLinkConfig,
-                        }}
-                      />
-                    ),
-                  },
-                ]}
-              />
-            </SectionLayout>
-          </>
-        ),
+        lazy: () => import('@/pages/Home').then(convertModuleToRouteObject),
       },
       {
         path: 'Challenges/DetailsPage',
-        element: <ChallengeDetailsPage />,
-        children: challengeDetailsPageTabRoutes,
+        lazy: () =>
+          import('@/pages/ChallengeDetailsPage').then(
+            convertModuleToRouteObject,
+          ),
+        children: [
+          {
+            index: true,
+            element: <RedirectWithQuery to={OVERVIEW_PATH} />,
+          },
+          {
+            path: OVERVIEW_PATH,
+            lazy: () =>
+              import('@/pages/ChallengeDetailsPageTabContent/OverviewTab').then(
+                convertModuleToRouteObject,
+              ),
+          },
+          {
+            path: INSTRUCTIONS_PATH,
+            lazy: () =>
+              import(
+                '@/pages/ChallengeDetailsPageTabContent/InstructionsTab'
+              ).then(convertModuleToRouteObject),
+          },
+          {
+            path: NEWS_PATH,
+            lazy: () =>
+              import('@/pages/ChallengeDetailsPageTabContent/NewsTab').then(
+                convertModuleToRouteObject,
+              ),
+          },
+          {
+            path: COMMUNITY_PATH,
+            lazy: () =>
+              import(
+                '@/pages/ChallengeDetailsPageTabContent/CommunityTab'
+              ).then(convertModuleToRouteObject),
+          },
+          {
+            // Task tab last so we attempt to match the other paths first before using the slug
+            path: ':taskId',
+            lazy: () =>
+              import('@/pages/ChallengeDetailsPageTabContent/TaskTab').then(
+                convertModuleToRouteObject,
+              ),
+          },
```

Comment:
Before, we hard-coded `Task1`, `Task2`, etc. as individual routes. Instead, use one route with a path parameter.